packages/create-llama/templates/components/use-cases/typescript/deep_research/src/app/workflow.ts

```diff
@@ -361,7 +363,7 @@ const createResearchPlan = async (
     context_str: contextStr,
     conversation_context: conversationContext,
     enhanced_prompt: enhancedPrompt,
-    user_request: userRequest,
+    user_request: userRequest as string,
```

Comment:
usually a bad idea to use `as` just to make TS happy
```suggestion
    user_request: extractText(userRequest),
```